bin/reth/src/version.rs

```diff
@@ -1,13 +1,34 @@
-//! This module contains the version message for the program.
-include!(concat!(env!("OUT_DIR"), "/built.rs"));
+//! Version information for reth.
 
-const VERSION: &str = PKG_VERSION;
-const NAME: &str = PKG_NAME;
-const SHA: Option<&str> = GIT_COMMIT_HASH_SHORT;
-const OS: &str = std::env::consts::OS;
+/// The short version information for reth.
+///
+/// - The latest version from Cargo.tom
+/// - The short SHA of the latest commit.
+///
+/// Example: "1.2.3 (defa64b2)"
+pub(crate) const SHORT_VERSION: &str =
+    concat!(env!("CARGO_PKG_VERSION"), " (", env!("VERGEN_GIT_SHA"), ")");
 
-/// The version message for the current program, like
-/// `reth/v0.1.0/macos-6fc95a5/`
-pub fn version_message() -> String {
-    format!("{}/v{}/{}-{}", NAME, VERSION, OS, SHA.unwrap())
-}
```

Comment:
I know this isn't used anywhere, because I didn't integrate this yet, but this was intended for the client version that's being used in the p2p messages, which should also be customizable via cli arg.

this is not a dealbreaker for this PR, I'll just open an issue for this separately.